backend-api/build.gradle

```diff
@@ -17,6 +17,9 @@ version '1.0-SNAPSHOT'
 
 sourceCompatibility = 1.11
 
+bootJar {
+    mainClassName = 'pms.PetMgtSystemApplication'
+}
```

Comment:
hmm this shouldn't be needed AFAIK

Reply:
We can remove it, the build result was the same with or without it.

Reply:
https://docs.spring.io/spring-boot/docs/2.2.0.RELEASE/gradle-plugin/reference/html/#packaging-executable-jars

> Executable jars can be built using the bootJar task. The task is automatically created when the java plugin is applied and is an instance of BootJar. The assemble task is automatically configured to depend upon the bootJar task so running assemble (or build) will also run the bootJar task.

We are using the `java` plugin so `gradle build` should be generating the jar file already.